### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by Bala667**
Hi Norman,

I have updated the GitHub example of SLMP connector please review it and let me know the comments thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,36 +27,36 @@ These PLC variants are supported:
...
+![overview](/docs/graphics/Slmparchi.png)
 
 ## Requirements
 
 ### Prerequisites
 
 * Access to an Industrial Edge Management (IEM) with onboarded Industrial Edge Device (IED)
 * IEM: Installed system configurator for Databus
-* IED: Installed apps IE Databus, IE Flow Creator, IIH Configurator, IIH Registry Service, SLMP Connector, Data Service (optional)
+* IED: Installed apps Databus, Flow Creator, common Configurator, Registry Service, SLMP Connector, IIH Essentials (optional)
```
Comment (HummelN7872) [inline README.md:44]: Please write "Common" instead of "common"

File: docs/graphics/SLMPdeploy.PNG
Comment (HummelN7872) [inline docs/graphics/SLMPdeploy.PNG:1]: Is there a reason why Tag1 and Tag 3 have a capital "T" and tag2 has a smal "t"

Comment (Bala667) [reply docs/graphics/SLMPdeploy.PNG:1]: Hi Norman,

I didn't use that image, So I will delete that

File: README.md (hunk 1 above)
Comment (Bala667) [reply README.md]: I changed this

**Review by HummelN7872 — APPROVED** [review]

**Pull request merged**